custom_components/nordpool/misc.py

```diff
@@ -26,29 +28,30 @@
 _LOGGER = logging.getLogger(__name__)
 
 
-def predicate(value):
+def predicate(value: Union[bool, None]) -> bool:
```

Comment:
Type hint. I see only bool output here.

Reply:
The output can only be bool, but the input might be None, True, False